README.md

```diff
@@ -1,4 +1,4 @@
-# Async.js
+![Async Logo](https://raw.githubusercontent.com/caolan/async/master/logo/async-logo_readme.jpg)
```

Comment:
That doesn't work... we need to access `raw.githubusercontent.com/...`, doing a relative path access, will access `github.com/caolan/logo/async-logo_readme.jpg` witch is an `404`...